orca-core/src/main/java/com/netflix/spinnaker/orca/config/RedisConfiguration.java

```diff
@@ -77,22 +91,25 @@ JedisPool jedisPoolPrevious(
     return createPool(null, previousConnection, timeout, registry, "jedisPoolPrevious");
   }
 
+  @Deprecated // rz - Kept for backwards compat with old connection configs
   @Bean(name = "redisClientDelegate")
   @Primary
   RedisClientDelegate redisClientDelegate(
     @Qualifier("jedisPool") Pool<Jedis> jedisPool
   ) {
-    return new JedisClientDelegate(jedisPool);
+    return new JedisClientDelegate("primaryDefault", jedisPool);
```

Comment:
Setting the old client delegates with `primaryDefault` and `primaryPrevious` is necessary for `RedisClientSelector` to be able to fallback to correctly if the desired named clients are not configured.